src/deadline/cinema4d_submitter/cinema4d_render_submitter.py

```diff
@@ -244,10 +246,12 @@ def _show_submitter(parent=None, f=Qt.WindowFlags()):
     # Set the setting defaults that come from the scene
     render_settings.name = Path(Scene.name()).name
     render_settings.frame_list = str(Animation.frame_list())
-    render_settings.output_path = Scene.output_path()
+    default_path, multi_path = Scene.get_output_paths()
+    render_settings.output_path = default_path
+    render_settings.multi_pass_path = multi_path
 
     # Load the sticky settings
-    render_settings.load_sticky_settings(Scene.name())
+    # render_settings.load_sticky_settings(Scene.name())
```

Comment:
Hmm, any reason we commented this line out? Is it not required anymore?

Reply:
My mistake, thanks for catching this

Reply:
No problem.